README.md

```diff
@@ -18,27 +20,26 @@ This repository is available to make related open source components accessible e
 
 ## Download the Latest Releases
```

Comment:
The "Download the Latest Releases" header should be a subheader of the "Related Open Source Projects" header. (unless you intend for it to be a link to the Microsoft.Build.Sql.Templates release.